pkg/reconciler/certificate/certificate.go

```diff
@@ -104,6 +108,11 @@ func (c *Reconciler) reconcile(ctx context.Context, knCert *v1alpha1.Certificate
 	}
 
 	knCert.Status.NotAfter = cmCert.Status.NotAfter
+	renewed := false
+	if cmCert.Status.RenewalTime != nil && time.Now().After(cmCert.Status.RenewalTime.Time) {
```

Comment:
Could you add a comment line for this with your issue link https://github.com/knative-sandbox/net-certmanager/issues/416 ?
I think this issue is difficult to notice and understand.